app/templates/find_a_legal_adviser/search.html

```diff
@@ -42,7 +42,14 @@ <h2 class="govuk-heading-m">What happens next</h2>
                 "classes": "govuk-input--width-10 postcode-search",
             } ) }}
 
-            {{ form.category }}
+            {% if form.category.data %}
+                {{ form.category }}
+            {% else %}
```

Comment:
I think you can remove this empty `{% else %}`

Reply:
Updated!